dynamicProgramming/CoinChange.java

```diff
@@ -35,9 +35,6 @@
  * The base case is that for making amount zero, we need Zero coins.
  * 
  * We can define a dp array and initialize this as the base case. For all other index, lets first assume that the amount cannot be given with the coins and initialize them as the maximum number, i have used 10001 as the amount can be anthing in range 0 to 10000.
- * We will try to solve this using tabulation using a dp array and a max variable to store the max length among all increasing sequences.
```

Comment:
Is this intentional?

Reply:
@roshinjojuc Is this intentional?

Reply:
no this must be a copy edit mistake, will remove it now